src/app.hooks.js

```diff
@@ -118,6 +119,30 @@ const displayInternRequests = (level) => (context) => {
 	return context;
 };
 
+/**
+ * For errors without error code create GeneralError with code 500.
+ * @param {context} context
+ */
+const errorHandler = (context) => {
+	if (context.error) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must not be padded by blank lines. (padded-blocks)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4527512)